XiEditor/EditView.swift

```diff
@@ -500,168 +491,16 @@ class EditView: NSView, NSTextInputClient {
         return (lineIx, col)
     }
 
-    /*  Insertion point blinking.
-        Only the frontmost ('key') window should have a blinking insertion point.
-        A new 'on' cycle starts every time the window is comes to the front, or the text changes, or the ins. point moves.
-        Type fast enough and the ins. point stays on.
-     */
-
-    /// Turns the ins. point visible, and set it flashing. Does nothing if window is not key.
-    func setInsertionBlink(_ on: Bool) {
-        // caller must set NeedsDisplay
-        cursorFlashOn = on
-        blinkTimer?.invalidate()
-        if on {
-            blinkTimer = Timer.scheduledTimer(timeInterval: TimeInterval(1.0), target: self, selector: #selector(blinkInsertionPoint), userInfo: nil, repeats: true)
-        }
-        else {
-            blinkTimer = nil
-        }
-    }
-
-    // Just performs the actual blinking.
-    func blinkInsertionPoint() {
-        cursorFlashOn = !cursorFlashOn
-        needsDisplay = true
-    }
-
-    // Current color for the ins. point. Implements flashing.
-    func cursorColor() -> CGColor {
-        if cursorFlashOn {
-            return CGColor(gray: 0, alpha: 1) // Black
-        }
-        else {
-            return CGColor(gray: 1, alpha: 1) // should match background.
-        }
-    }
-
-    // Background color for selected text. Only the first responder of the key window should have a non-gray selection.
-    func selcolor() -> NSColor {
```

Comment:
This PR is not compiling for me, selcolor() is still being referenced from EditView.draw().